packages/features/users/components/UserTable/UserListTable.tsx

```diff
@@ -186,6 +186,10 @@ export function UserListTable() {
         id: "select",
         enableHiding: false,
         enableSorting: false,
+        meta: {
+          sticky: true,
+          stickLeft: 0,
+        },
```

Comment:
Allows us to have sticky and offset the left column depending on what column is being used

Reply:
Quick Q: Would this affect RTL-LTR layout differences?